lib/SeqFromQuery.pm

```diff
@@ -31,8 +31,6 @@ use Statistics::Distributions qw(chisqrdistr udistr);
 
 use Math::Round qw/nhimult round/;
 
-use POSIX qw/lround/;
```

Comment:
This library should be deleted; obviated by search/save; the corresponding Elasticsearch 5 libraries should also be removed from install-perl-libs.sh

Reply:
Also need to make sure we support the additional delimiter in search/save